src/libraries/Microsoft.PowerFx.Core/Parser/ParserOptions.cs

```diff
@@ -20,18 +20,39 @@ public class ParserOptions
         /// </summary>
         public bool AllowsSideEffects { get; set; }
 
+        /// <summary>
+        /// The culture that an expression will parse with. 
+        /// This primarily determines numeric decimal separator character
+        /// as well as chaining operator. 
```

Comment:
any impact on date formatting at this stage?

Reply:
This shouldn't have any behavior change to existing behavior.   (Although we will want to tackle that separately for our outstanding locale bugs like #990)